util.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const cloudRunServiceNameRandSuffixLen = 10
+
+// gcloudCommandBuildcreates an exec.Cmd that calls the external gcloud executable. The exec.Cmd's name is set to
+// "gcloud" and the args are the ones provided in addition to a project flag and quiet flag.
+func gcloudCommandBuild(arg []string) *exec.Cmd {
```

Comment:
I think every instance of this function is wrapped with `execCommand` - does it really make sense for those to be two separate functions?

Reply:
Every instance of gcloudCommandBuild is not wrapped with execCommand. https://github.com/GoogleCloudPlatform/serverless-sample-tester/pull/1/files#diff-95256f97482a6d6739fe6ee6897b4114R63 is an example. And vice versa, every instance of execCommand doesn't wrap gcloudCommandBuild. https://github.com/GoogleCloudPlatform/serverless-sample-tester/pull/1/files#diff-95256f97482a6d6739fe6ee6897b4114R30 is an example. So for right now, I think it makes sense to keep these two functions separate.